python/lsst/obs/base/script/convertGen2RepoToGen3.py

```diff
@@ -30,9 +30,9 @@
 import lsst.log
 import lsst.utils
 
-from lsst.obs.base.gen2to3 import (ConvertRepoTask, ConvertRepoSkyMapConfig,
-                                   Translator, ConstantKeyHandler, CopyKeyHandler,
-                                   CalibKeyHandler)
+from ..gen2to3 import (ConvertRepoTask, ConvertRepoSkyMapConfig,
+                       Translator, ConstantKeyHandler, CopyKeyHandler,
+                       CalibKeyHandler)
```

Comment:
Not worth doing now, but I've found I really like just using line breaks between each symbol when it gets longer than a single line - much less churn when new ones are adding, and easier diffs as a result.